### ISSUE TITLE
[peerflix auto-play problem] Cannot read property ProgramPath of undefined

### ISSUE THREAD

**Issue by SMed79** [mafintosh/peerflix#194]
Hello,
on windows i can not auto-play using "--potplayer" option !
i have an error message related with ProgramPath : 
```
C:\Users\SMed79\cd Desktop
C:\Users\SMed79\Desktop>peerflix test.torrent --potplayer

C:\Users\SMed79\AppData\Roaming\npm\node_modules\peerflix\app.js:241
	if (!key['ProgramPath'])

TypeError: Cannot read property 'ProgramPath' of undefined
	at Server.<anonymous> (C:\Users\SMed79\AppData\Roaming\npm\node_modules\peerflix\app.js:241:15)
	at Server.emit (events.js:104:17)
	at net.js:1144:12
	at process._tickCallback (node.js:355:11)

C:\Users\SMed79\Desktop>
```

any idea?

**Comment by jaruba** [comment]
Well, from what I see the implementation for pot player is pretty basic: https://github.com/mafintosh/peerflix/commit/3853340095062649d1b89fdbcdd042f9cfa4520d

So it takes the path from the registry at:

      if (process.arch === 'x64')
        key = registry('HKCU/Software/DAUM/PotPlayer64')

      if (!key['ProgramPath'])
        key = registry('HKCU/Software/DAUM/PotPlayer')

So the question is, if you go to Start, then in the Search Box write "regedit" and hit enter. And go to ```HKEY_CURRENT_USER\Software\DAUM``, do you see any ``PotPlayer`` or ``PotPlayer64`` or can you visually locate something related to PotPlayer anywhere in the registry there?

If you find it, then click it and see if it has the ``ProgramPath`` key with the exact location of the PotPlayer executable.

**Comment by SMed79** [comment]
I think i dont have the correct PotPlayer "ProgramPath" on windows registry !
```
[HKEY_CURRENT_USER\Software\DAUM\PotPlayer]
"ProgramFolder"="C:\\Program Files\\DAUM\\PotPlayer"
"ProgramPath"="C:\\Program Files\\DAUM\\PotPlayer\\PotPlayerMini.exe" 
```

The "--vlc" option working fine without any problem.

**Comment by jaruba** [comment]
So you have "PotPlayer" in the registry, do you have a 32bit or 64bit OS?

**Comment by jaruba** [comment]
Also, as a test, could you try opening app.js with a text editor, and changing the line at 241 from:

    if (!key['ProgramPath'])

to

    if (typeof key === 'undefined')

then trying to launch peerflix with pot player again?

**Comment by SMed79** [comment]
I have a 32bit OS & i confirm the change solve the problem and launch "PotPlayer" auto-play using "--potplayer" option.

![peerflix-app-js](https://cloud.githubusercontent.com/assets/1873139/6734676/db33ebb0-ce59-11e4-85e1-cbf16a5d1f6a.png)

@jaruba Thank you very much for your help.

**Comment by jaruba** [comment]
Glad to hear it works :)

**Comment by mafintosh** [comment]
Super. Fix for this out in 0.29.1
